cli/man/splinter.1.md

```diff
@@ -0,0 +1,105 @@
+% SPLINTER(1) Cargill, Incorporated | Splinter Commands
+
+NAME
+====
+
+**splinter** — Command-line interface for Splinter
+
+SYNOPSIS
+========
+
+**splinter** \[**FLAGS**\] \[**SUBCOMMAND**\]
+
+DESCRIPTION
+===========
+
+The `splinter` utility is the command-line interface for Splinter, a
+privacy-focused platform for distributed applications. This CLI provides a
+unified tool for creating circuits, generating keys, and other functions for
+managing a Splinter node.
+
+* Run `splinter --help` to see the list of subcommands.
+
+* Run `splinter *SUBCOMMAND* --help` to see information about a specific
+  subcommand.
+
+* To view the man page for a Splinter subcommand, use the "dashed form" of the
+  name, where each space is replaced with a hyphen. For example, run
+  `man splinter-circuit-list` to see the man page for `splinter circuit list`.
+
+SUBCOMMANDS
+===========
+
+`admin`
+: Provides administrative functions with the `keygen` and `keyregistry`
+  subcommands
+
+`cert`
+: Generates insecure certificates for development with the `generate`
+  subcommand
+
+`circuit`
+: Provides circuit creation and management functions with `list`, `propose`,
+  `template`, `vote`, and other subcommands
+
+`database`
+: Provides database functions with the `migrate` subcommand
+
+`health`
+: Displays information about network health with the `status` subcommand
+
+`keygen`
+: Generates secp256k1 user keys.
```

Comment:
`keygen` is part of `admin`

Reply:
Actually, we are moving it down a level and getting rid of admin.